docs/scss/_docs-general.scss

```diff
@@ -72,6 +74,45 @@ hr{
     margin: 36px 0;
 }
 
+pre {
```

Comment:
Is this duplication on purpose? https://github.com/SAP/fundamental/pull/1216/files#diff-6f870081dc262292999a86c784066906R8848

Reply:
This is the scss source file. the other file is the compiled css file of the same code so it's not duplication.